tests/unit/evaluation/SplittingStrategy_unittest.cc

```diff
@@ -296,3 +296,90 @@ TEST(SplittingStrategy,LOO)
 		SG_UNREF(splitting);
 	}
 }
+
+TEST(SplittingStrategy, timeseries_subset_linear_splits)
+{
+	index_t num_labels, num_subsets, min_subset_size, base_size;
+	index_t runs = 10;
+
+	while (runs-- > 0)
+	{
+		num_labels = CMath::random(50, 150);
```

Comment:
Could you create a mock class here, or at least a function that generates all the code that is shared across the different tests? See e.g. `tests/unit/FisherLDA_unittest.cc` on how to do that

Reply:
I saw the FisherLDA_unittest.cc, great way. I will do like that.